src/ua/hillel/Cat.java

```diff
@@ -0,0 +1,24 @@
+public class Cat extends Animals {
+    int max_run = 200;
+    int max_swim = 0;
+    int add_cat;
```

Comment:
`add_cat` variable should count only cats. In your case, it seems, that if you create 2 Cat objects, `add_cat` will be equal to 1 for both cats. So, your task here is to calculate how many cats were created overall during the work of your program. Make it private and static.